catalog/additionalInfo.md

```diff
@@ -0,0 +1,15 @@
+# Using GitHub Personal Access Tokens
+
+Integration with GitHub requires the user to create or use a private
+access token as a one-time setup step. This GitHub personal access token
+increases security, because it ensures each user can only see repositories
+they have access to. New Relic will store that access token, but the user
+can easily delete it from New Relic's storage at any time.
+
+We recommend creating a dedicated personal access token for this application only,
+and you should give that token `read only` permissions. See
+[GitHub's doc on access tokens](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line)
+for full instructions.
+
+Personal access tokens are stored with New Relic One's `UserStorage`
+APIs. `UserStorage` data is accessible only to the user account and `Nerdpack` which created it. The data is encrypted in transmission, but is not encrypted at rest (within the underlying database). Because of this, `UserStorage` data may be accessible to a New Relic employee via an administrative privilege in the platform.w
```

Comment:
I think there's an extra "w" here at the end.

```suggestion
APIs. `UserStorage` data is accessible only to the user account and `Nerdpack` which created it. The data is encrypted in transmission, but is not encrypted at rest (within the underlying database). Because of this, `UserStorage` data may be accessible to a New Relic employee via an administrative privilege in the platform.
```

Reply:
Removed